tests/unit/events.spec.ts

```diff
@@ -43,8 +43,8 @@ describe("EventEmitter", () => {
             expect(event).toEqual(args);
             done();
         };
-        emitter.addListener(args.type, callback);
-        emitter.emit(args.type, args);
+        emitter.addListener(args.name, callback);
+        emitter.emit(args.name, args);
```

Comment:
This has been the wrong property name for a little while, the tests passed because it was just matching on undefined.

Reply:
LOL